kafka-client/src/main/resources/otel-jmx.config.yaml

```diff
@@ -347,3 +347,88 @@ rules:
        type: gauge
        desc: total records processed
        unit: '{records}'
+
+ # MongoDB Client Metrics
+ - bean: dev.responsive:type=mongodb,command=*
```

Comment:
Good chance to get naming consistent. Perhaps the type could be "mongodb-client" and "cassandra-client"?

Reply:
Yeah, that's what I was thinking too. Will update.